android-gif-drawable/src/main/java/pl/droidsonroids/gif/GifDecoder.java

```diff
@@ -150,6 +150,13 @@ public boolean isAnimated() {
 		return mGifInfoHandle.getNumberOfFrames() > 1 && getDuration() > 0;
 	}
 
+	/**
+	 * @return true if GIF is opaque, false otherwise
```

Comment:
It is not necessarily correct.
Opacity here is not calculated during actual file decoding.
This value can only be set here by the caller: https://github.com/koral--/android-gif-drawable/blob/dev/android-gif-drawable/src/main/java/pl/droidsonroids/gif/GifOptions.java#L58

I don't see any value in getting back.

What is your intent behind this change?

Reply:
I want to get the value from GifDecoder.Pls abandon the pr if the value not right.